Formula/tesseract.rb

```diff
@@ -92,6 +98,9 @@ def install
     system "./configure", *args
 
     system "make", "install"
+    if build.with? "serial-num-pack"
+      resource("snum").stage{ mv "snum.traineddata", share/"tessdata" }
```

Comment:
It's probably alright; `tessdata` isn't a particularly generic name & it's not placing any _files_ in the top-level prefix.